tools/header_guard_check/lib/header_guard_check.dart

```diff
@@ -45,14 +48,20 @@ final class HeaderGuardCheck {
   /// Path directories to exclude from the check.
   final List<String> exclude;
 
+  /// Stdout.
+  final StringSink _stdOut;
+
+  /// Stderr.
+  final StringSink _stdErr;
```

Comment:
pedantic nit: why not just `_stdout`, `_stderr` to match capitalisation elsewhere, and in the dart IO APIs?

Reply:
Because I hate myself I suppose.